### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by prosoitos**
I think that DataFrame and not vector was meant in this sentence.

### CODE REVIEW

**Comment by bkamins** [comment]
The previous wording was correct. What is meant that vector `[:A]` in this case contains one column but in general it could contain more.

However, if you find the current wording unclear please go ahead and correct the sentence (I am not a native so ...).

File: docs/src/man/getting_started.md [hunk 1]
```diff
@@ -419,7 +419,7 @@ julia> df[!, :A] == df[:, :A]
 true
 ```
 
-In the first cases, `[:A]` is a vector, indicating that the resulting object should be a `DataFrame`, since a vector can contain one or more column names. On the other hand, `:A` is a single symbol, indicating that a single column vector should be extracted. Note that in the first case a vector is required to be passed (not just any iterable), so e.g. `df[:, (:x1, :x2)]` is not allowed, but `df[:, [:x1, :x2]]` is valid.
+In the first cases, `[:A]` is a vector, indicating that the resulting object should be a `DataFrame`, since a DataFrame can contain one or more column names. On the other hand, `:A` is a single symbol, indicating that a single column vector should be extracted. Note that in the first case a vector is required to be passed (not just any iterable), so e.g. `df[:, (:x1, :x2)]` is not allowed, but `df[:, [:x1, :x2]]` is valid.
```
Comment (bkamins) [inline docs/src/man/getting_started.md:422]:
```suggestion
In the first case, `[:A]` is a vector, indicating that the resulting object should be a `DataFrame`. On the other hand, `:A` is a single symbol, indicating that a single column vector should be extracted. Note that in the first case a vector is required to be passed (not just any iterable), so e.g. `df[:, (:x1, :x2)]` is not allowed, but `df[:, [:x1, :x2]]` is valid.
```

**Comment by bkamins** [comment]
Thank you for your suggestion. I have removed the confusing part in the end as it was not needed (and only caused a distraction).

**Comment by prosoitos** [comment]
Great. Thanks!

**Pull request merged**